app/models/course/discussion/topic.rb

```diff
@@ -10,6 +10,17 @@ class Course::Discussion::Topic < ActiveRecord::Base
 
   accepts_nested_attributes_for :posts
 
+  # Topics to be displayed in the comments center.
+  scope :to_display, (lambda do
```

Comment:
I know it violates the open/close principle and I actually had thought about that: asking sub classes to declare that they should be displayed. Then the implementation will look like:
```
   where(actable_type: subclasses.select { |t| t.display_in_comments_center? })
```

But I checked the acts-as gem, there is no way to get all the subclasses, then we will need to do something similar to what condition did: https://github.com/Coursemology/coursemology2/blob/master/app/models/course/condition.rb#L10, which is not better IMO, cause the parent class still need to be changed when there is a new subclass. 

So finally I just used the simplest way as you can see here, what do you think ?